gsa/src/gmp/models/cve.js

```diff
@@ -21,13 +21,9 @@ import {isArray, isDefined} from 'gmp/utils/identity';
 import {isEmpty} from 'gmp/utils/string';
 import {map} from 'gmp/utils/array';
 
-import {
-  parseSeverity,
-  parseCvssBaseVector,
-  parseDate,
-  setProperties,
-} from 'gmp/parser';
+import {parseSeverity, parseDate, setProperties} from 'gmp/parser';
 
+import {parseCvssV2BaseVector} from 'gmp/utils/calculator';
```

Comment:
Would be better to put the functions into `gmp/parser/cvss.js` Yes they are used for the calculator but looking only at the functions they are used to parse cvss vectors only.

Reply:
Is it possible to have an directory with the same name as a file?

Reply:
it is very similar as in Python. You can choose a index.js file which acts like the __init__.py file. E.g. https://github.com/greenbone/gsa/blob/master/gsa/src/gmp/locale/index.js where I split the locale.js file into several files and wanted to keep backwards compatibility with existing imports.